compiler/rustc_codegen_llvm/src/gotoc/rvalue.rs

```diff
@@ -656,11 +656,25 @@ impl<'tcx> GotocCtx<'tcx> {
                 let src_goto_expr = self.codegen_operand(o);
                 let src_mir_type = self.operand_ty(o);
                 let dst_mir_type = t;
-                let dst_goto_expr = self.cast_sized_expr_to_unsized_expr(
-                    src_goto_expr.clone(),
-                    src_mir_type,
-                    dst_mir_type,
-                );
+
+                // Check if the cast is from a vtable fat pointer to another
+                // unsized type (which can happen with auto trait fat pointers)
+                let possible_dyn_src = dyn_from_vtable_fat_pointer(src_mir_type);
```

Comment:
do you use this? If not, maybe just move it to the next line

Reply:
Done!